dashboard/dashboard-user.md

```diff
@@ -0,0 +1,33 @@
+---
+title: TiDB Dashboard 用户
+aliases: ['/docs-cn/dev/dashboard/dashboard-user/']
+---
+
+# TiDB Dashboard 用户
+
+从 TiDB 5.3 开始，TiDB Dashboard 支持使用任意 SQL 用户登录，该 SQL 用户必须拥有以下权限：
```

Comment:
Add a new section `example` below.

Reply:
How about calling it "customized" instead of "arbitrary"? Arbitrary means any, but it is not precise.

```suggestion
从 TiDB 5.3 开始，TiDB Dashboard 支持使用自定义的 SQL 用户登录访问。该 SQL 用户必须拥有以下权限：
```